components/has/kustomization.yaml

```diff
@@ -1,10 +1,15 @@
 resources:
 - allow-argocd-to-manage.yaml
-- catalogsource.yaml
-- operatorgroup.yaml
-- subscription.yaml
+- https://github.com/redhat-appstudio/application-service/config/default
 - .tekton/
 
 
 apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
\ No newline at end of file
+kind: Kustomization
+
+images:
+- name: quay.io/redhat-appstudio/application-service
+  newName: quay.io/redhat-appstudio/application-service
+  newTag: sha-9b9dcc9
```

Comment:
This is what will be bumped again and again.

Reply:
Can (and/or should?) we point to a specific ref in https://github.com/redhat-appstudio/application-service too?